Bugfix on relative BASE_URL

Pe5h4 opened the pull request:
- Added check on relative `BASE_URL` - if found, `BASE_URL` is set to browser's URL.
- Added check on absolute `API_PATH` - if found, `API_PATH` is used as a `service_url`
- Removed `hotfix` from https://github.com/TeskaLabs/asab-webui/commit/90f68f5100beea5c4a3358651f0c68ee284d8d9a

ateska commented on src/containers/Application.js:
```diff
@@ -179,15 +179,26 @@ it is accessible by the sidebar toggler button.
 			BASE_URL = this.Config.get('BASE_URL');
+		} else {
+			// If BASE_URL has been defined but with relative path => take browser URL as BASE_URL
+			BASE_URL = window.location.protocol + '//' + window.location.host + window.location.pathname.replace(/\/$/, '');
```
```
+ this.Config.get('BASE_URL');
```

?

Otherwise it feels contra-intuitive.

Pe5h4 commented:
@ateska I added the appending of relative `BASE_URL` behind the browser URL. I also added a check on trailing slashes when relative `BASE_URL` or `API_PATH` are set since we append trailing slashes automatically when composing `service_url`. 

https://github.com/TeskaLabs/asab-webui/blob/053ec02398195077c935f00c9a7ab689aa7561b0/src/containers/Application.js#L191

https://github.com/TeskaLabs/asab-webui/blob/053ec02398195077c935f00c9a7ab689aa7561b0/src/containers/Application.js#L209

Now the relative BASE_URL and API_PATH can be set as following:

```
BASE_URL: '/relative/path',
API_PATH: '/api/path',
```
or

```
BASE_URL: 'relative/path',
API_PATH: 'api/path',
```
Regardless if there is a trailing slash in the first position of the BASE_URL or API_PATH or not.
